tools/targets/ARM_MUSCA_A1/ARM_MUSCA_A1_TFM.py

```diff
@@ -0,0 +1,103 @@
+ #
+ # Copyright (c) 2018 Arm Limited
+ #
+ # Licensed under the Apache License Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ #     http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing software
+ # distributed under the License is distributed on an "AS IS" BASIS
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+ #
+
+try:
+ import Crypto
+except ImportError, e:
+ print('Please install "pycryptodome" Python module! ("pip install --user pycryptodome")')
+ quit()
+
+import sys
+import os.path
+from os.path import(
+    join,
+    dirname
+)
+run_dir = dirname(os.path.realpath(__file__))
+sys.path.append(run_dir)
+
+import assemble
+import imgtool
```

Comment:
Could you import these modules (which I assume are the local ones) using a relative import?